src/components/radio/RadioSelectReducers.ts

```diff
@@ -0,0 +1,29 @@
+import * as _ from 'underscore';
+import {IReduxAction} from '../../utils/ReduxUtils';
+import {IRadioSelectActionPayload, RadioSelectActions} from './RadioSelectActions';
+
+export interface IRadioSelectState {
+    id: string;
+    value: string;
+    disabledValues: string[];
+}
+
+export const radioSelectInitialState: IRadioSelectState = {id: undefined, value: undefined, disabledValues: undefined};
+export const radioSelectsInitialState: IRadioSelectState[] = [];

```

Comment:
Any reason to use an array instead of an object ?

Reply:
this should also be a group decision, it can get messy if we start to go toward the Map strategy instead of array,